handler/aws.go

```diff
@@ -27,6 +27,15 @@ func init() {
 		host := fmt.Sprintf("execute-api.%s.amazonaws.com", region)
 		services[host] = endpoints.ResolvedEndpoint{URL: fmt.Sprintf("https://%s", host), SigningMethod: "v4", SigningRegion: region, SigningName: "execute-api"}
 	}
+
+	// Add s3 endpoints
+	for region := range endpoints.AwsPartition().Regions() {
```

Comment:
do you have a use case that prevents you from using `s3.<region>.amazonaws.com`?